packages/core-data/src/entity-types/settings.ts

```diff
@@ -90,6 +90,10 @@ declare module './base-entity-records' {
 			 * Site icon.
 			 */
 			site_icon: number;
+			/**
+			 * The ID of the page that should display posts.
+			 */
+			page_for_posts: number;
```

Comment:
I've just added this yesterday as part of https://github.com/WordPress/gutenberg/commit/5f29581507b51dc6505494795b275b41ad09be69 so that's why the actions are failing (duplicate identifier).